### PR TITLE
Updated to readme schematic images

### PR DESCRIPTION
**Pull request by rosscullen**
Hi @adonno 
Here's my small contribution back for all your help. 
Updated Fritzing diagram and PNG image on readme - new layout to match the positioning in the case. Labels, DIN etc all uploaded based on the one I received. Also inverted tc/rx as they appeared incorrectly (as per my recent discord message).
Also included an image of my completed layout to match the schematic as best as possible.

Hope its of use. I could as some FAQ as well if its of help to readme? EG adding switched to Lovelace to control LED, buzzer etc. Also getting rid of the red LED on reader.

Thanks. R

### CODE REVIEW

**Review by jesserockz — CHANGES_REQUESTED** [review]
You seem to have the buzzer wired up to `5v` and `GND` in your schematic, but it should be `D7` and `GND`?

**Comment by rosscullen** [comment]
> You seem to have the buzzer wired up to `5v` and `GND` in your schematic, but it should be `D7` and `GND`?

Will take a look @jesserockz thanks for the feedback

**Comment by rosscullen** [comment]
Thanks for feedback @jesserockz you were right on the money. Please can review again and see if we're good to go! Thanks :-)

**Comment by adonno** [comment]
D1 and D2 inverted ? 
D1 should go to  SCL
and D2 to SDA

**Comment by rosscullen** [comment]
> D1 and D2 inverted ?
> D1 should go to SCL
> and D2 to SDA

Thanks @adonno changes made. FZZ file and PNG updated. Also added updated labels on LED Board.

**Comment by adonno** [comment]
looks fine for me, @magnusoverli  @jesserockz does one of you guys mind having one last look and merge if you are ok with it?

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Recent reports suggesting more clones of the D1 is now working as expected. This
 
 ### Connecting the components
 
-![Photo of schematics](Schematics/tag_reader_schematics_v2.png)
```
Comment (jesserockz) [inline README.md:29]: I know this is not exactly your fault @rosscullen as the repo was already set this way. 
But there is no point in creating version numbers for files in git... You should just be overriding the old file with a new one. Git is the version manager. cc @adonno @magnusoverli

**Pull request merged**